src/components/Navigation/SideDrawer/DrawerToggle/DrawerToggle.test.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import DrawerToggle from './DrawerToggle';
+
+describe('Render DrawerToggle', () => {
+  test('DrawerToggle element exists', () => {
+    render(<DrawerToggle />);
+    expect(screen.getByTestId('drawer-toggle')).toBeTruthy();
+  });
+
+  //   test('calls onClick when clicked', () => {
```

Comment:
instead of commenting you can add `test.skip` to `wip` tests, although it's not a good practice to have them on a codebase, you can contact any of the mentors if you need help with the test.